packages/cozy-harvest/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cozy-harvest",
+  "version": "0.1.0",
+  "description": "Harvest library for auto-apps in Cozy",
```

Comment:
Same here, I think we can provide a better description:

> Provides logic, modules and components for Cozy's harvest applications.

What do you think ?